docs/source/reference/object_detection.rst

```diff
@@ -0,0 +1,115 @@
+
+.. _object_detection:
+
+################
+Object Detection
+################
+
+********
+The task
+********
+
+Object Detection task helps to identify and locate objects in images.
+
+------
+
+*********
+Inference
+*********
+
+The :class:`~flash.vision.ObjectDetector` is already pre-trained on `COCO train2017 <https://cocodataset.org/>`_, a dataset with `91 classes <https://cocodataset.org/#explore>`_ (123,287 images, 886,284 instances).
+
+
+Use the :class:`~flash.vision.ObjectDetector` pretrained model for inference on any image tensor or image path using :func:`~flash.vision.ObjectDetector.predict`:
+
+.. code-block:: python
+
+	from flash.vision import ObjectDetector
+
+	# Load the model
+	detector = ObjectDetector()
+
+	# 2. Perform inference on an image file
+	predictions = detector.predict("path/to/image.png")
+	print(predictions)
+
+Or on a random image tensor
+
+.. code-block:: python
+
+    # 2. Perform inference on a random image tensor
+    import torch
+    images = torch.rand(32, 3, 1080, 1920)
+    predictions = detector.predict(images)
+    print(predictions)
+
+For more advanced inference options, see :ref:`predictions`.
+
+------
+
+**********
+Finetuning
+**********
+
+To tailor the object detector to your dataset, you would need to have it in `COCO Format <https://cocodataset.org/#format-data>`_, and then finetune the model.
```

Comment:
Can you explain what this means briefly?

Reply:
I think the link https://cocodataset.org/#format-data explains it well. We need metadata in this format for the images in the folder.

Reply:
@edenlightning

Reply:
it would be useful to add a short example, no need for details but give user idea...

Reply:
Added!